packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -173,6 +204,7 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
   @override
   void didUpdateWidget(TreeTable oldWidget) {
     super.didUpdateWidget(oldWidget);
+    sortData(widget.startingSortColumn, widget.startingSortDirection);
```

Comment:
nit: this is too aggressive a didUpdateWidget. Only update when the widget actually changed.